salt/iiif/loris.sls

```diff
@@ -42,31 +46,36 @@ loris-dependencies:
             - libfreetype6
             - libfreetype6-dev
             - zlib1g-dev
+            {% if osrelease == "14.04" %}
             - liblcms
             - liblcms-dev
             - liblcms-utils
+            - libtiff4-dev
+            {% endif %}
+
             - liblcms2-2 
             - liblcms2-dev 
             - liblcms2-utils
-            - libtiff4-dev
             - libtiff5-dev
             - libxml2-dev
             - libxslt1-dev
 
+    # TODO: these requirements best live in a requirements.txt file where we can get security feedback
     cmd.run:
         - name: |
+            set -e
             venv/bin/pip install Werkzeug==0.12.1
             venv/bin/pip install configobj==5.0.6
             venv/bin/pip install Pillow==4.1.0
-            venv/bin/pip install uwsgi==2.0.14
+            venv/bin/pip install uwsgi==2.0.17.1
```

Comment:
!important

without the `-e` I was getting a compilation failure when installing uwsgi and the cmd was otherwise returning correctly

[iiif.pipfail.txt](https://github.com/elifesciences/iiif-formula/files/2982012/iiif.pipfail.txt)

Reply:
probably because the last command exited successfully